### DISCUSSION TITLE
Support IDE Jetbrains

### DISCUSSION THREAD

**Discussion by davi19** [starship/starship#2842] [Q&A] [answered]
Hi, i'm using starship with fish with no problems, but in terminal of IDE of Jetbrains like Rider and Rubymine, the emojis not work. In config of terminal i have the font of nerdfotns. It's a bug or a config that is missing ?

![Captura de tela de 2021-07-01 09-31-40](https://user-images.githubusercontent.com/9946675/124125516-10dba100-da50-11eb-9509-0223fea0fe84.png)
**My terminal in IDE**

![Captura de tela de 2021-07-01 09-35-03](https://user-images.githubusercontent.com/9946675/124125536-1b963600-da50-11eb-9d05-e92797b8c4fa.png)
**Config of fonts of terminal**

![Captura de tela de 2021-07-01 09-35-19](https://user-images.githubusercontent.com/9946675/124125594-2f419c80-da50-11eb-90d5-d60d45bf8bbc.png)
**Config of terminal**

**Comment by davidkna** [comment]
Please try setting the fallback font to an emoji font like Noto Emoji instead of Fira Code.

**Reply by davi19** [reply]
i tried here but still no emoji.

**Comment by davi19** [comment, marked as answer]
is a bug in jetbrains IDEs, post to forums with the same error

https://youtrack.jetbrains.com/issue/IDEA-248010
